api/v1/urls.py

```diff
@@ -15,7 +15,7 @@
 _groups = 'groups'
 
 #: The URL namespace of the api.v1 app.
-app_name = 'api:v1'
+app_name = 'v1'
```

Comment:
Except as it was before such calls to reverse() literally did not work, the namespace was not being registered. Note that I changed this to just 'v1' since there is an enclosing namespace 'api' on the outer app.